vignettes/MSstatsUpdated.Rmd

```diff
@@ -0,0 +1,324 @@
+---
+title: "MSStats Vignette"
+date: September 5th, 2024
+---
+
+```{r style, echo = FALSE, results = 'asis'}
+BiocStyle::markdown()
+```
+
+```{r global_options, include=FALSE}
+knitr::opts_chunk$set(fig.width=10, fig.height=7, warning=FALSE, message=FALSE)
+options(width=110)
+```
+
+```{=html}
+<!--
+%\VignetteIndexEntry{MSstats: Protein/Peptide significance analysis}
+%\VignetteEngine{knitr::knitr}
+-->
+```
+# __MSstats: Protein/Peptide significance analysis__
+
+Package: MSstats
+
+Author: Anshuman Raina
+
+Date: 5th Semptember 2024
+
+## __Introduction__
+
+MSstats, an R package in Bioconductor, supports protein significance analysis for statistical relative quantification of proteins and peptides in global, targeted and data-independent proteomics. It handles shotgun, label-free and label-based (universal synthetic peptide-based) SRM (selected reaction monitoring), and SWATH/DIA (data independent acquisition) experiments. It can be used for experiments with complex designs (e.g. comparing more than two experimental conditions, or a time course).
+
+This vignette summarizes the introduction and various options of all functionalities in MSstats. More details are available in `User Manual`.
+
+## __Installation__
+
+To install this package, start R (version “4.0”) and enter:
+
+``` {r code Installation}
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+
+BiocManager::install("MSstats")
+library(MSstats)
+
+```
+
+## __1. Workflow__
+
+### __1.1 Raw Data__ 
+
+To begin with, we will load sample datasets, including both annotated and plain data. The dataset you need can be found [here](insert Link for dataset here).
```

Comment:
For the link, I'm tempted to provide the Github url - https://github.com/Vitek-Lab/MSstatsConvert/tree/devel/inst/tinytest/raw_data/PD

Reply:
Yeah, kept this as placeholder. Forgot to add this link.